### PR TITLE
py310

### PR DESCRIPTION
**Pull request by maxscheurer**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
Looks like the migration file is misformatted

File: .ci_support/migrations/python310.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+migrator_ts: 1634137107
+ __migrator:
```
Comment (beckermr) [inline .ci_support/migrations/python310.yaml]: Everything from here down is indented by one too many spaces.

Comment (maxscheurer) [reply .ci_support/migrations/python310.yaml]: Wooops... Copy/paste from GitHub web 🙈 Will fix it...

**Comment by maxscheurer** [comment]
@conda-forge-admin please rerender.

**Pull request merged**